### ISSUE TITLE
Create documentation for contributing

### ISSUE THREAD

**Issue by frederickalcantara** [OpenZeppelin/ethernaut#119]
We should create a proper guideline on how to properly contribute to this repo. This will allow anyone to get their feet wet and contribute accordingly.

**Comment by paulinablaszk** [comment]
Good point. Maybe we can adapt the Openzeppelin-solidity contribution guide 
https://github.com/OpenZeppelin/openzeppelin-solidity/blob/master/CONTRIBUTING.md

**Comment by frederickalcantara** [comment]
Another good format that I saw was over at prysmatic labs. [https://github.com/prysmaticlabs/prysm/blob/master/CONTRIBUTING.md]

**Comment by paulinablaszk** [comment]
I can prepare this contribution guide

**Comment by frangio** [comment]
Guidelines for adding a level are in the README. For any other doubts regarding contribution, please open an issue with your question.
